README.md

```diff
@@ -2,7 +2,11 @@
 
 The asynchronous web toolkit for clojure built on top of Ratpack
 
-[![Clojars Project](http://clojars.org/funcool/catacumba/latest-version.svg)](http://clojars.org/funcool/catacumba)
+[![Clojars Project](http://clojars.org/updcon/catacumba/latest-version.svg)](http://clojars.org/funcool/catacumba)
```

Comment:
This is untrue, so i'm not going to merge this PR with this text on the readme.

Reply:
Sure that untrue - I waited for a while for response and decided to clone the sorces since it is required for my other projects.
So, I have a question - do you have any plans to develop and to maintain this repo nearest time?
If yes - I'll happy to reverse these changes and prepare this patch for merge into the main tree.

Reply:
Please, i will gladly merge them and release a new version :D